src/routes/root.js

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+// This router implements a very simple route for /, so that we can more easily
+// test the server through tools like the Mozilla Observatory.
+
+import Router from '@koa/router';
+import { getLogger } from '../log';
+
+export function rootRoutes() {
+  const log = getLogger('routes.root');
+  const router = new Router();
+
+  router.get('/', async (ctx) => {
+    log.verbose('/');
+
+    ctx.body = 'OK';
```

Comment:
Should we provide any additional information about what this page is to a user trying to access the API?